src/Package/Impl/Plots/PlotContentProvider.cs

```diff
@@ -31,7 +31,9 @@ internal sealed class PlotContentProvider : IPlotContentProvider {
         private async void RSession_Connected(object sender, EventArgs e) {
             // Let the host know the size of plot window
             if (_lastPixelWidth >= 0 && _lastPixelHeight >= 0) {
-                await ApplyNewSize();
+                try {
+                    await ApplyNewSize();
+                } catch (OperationCanceledException) { } catch (MessageTransportException) { } catch (RException) { }
```

Comment:
I would like to appropriate that exception for failing evals sometime later. Right now `EvaluateAsync` never throws but rather just returns `REvaluationResult`, which you then have to check... and every place that uses it ends up doing that same check over and over again (or sometimes forget to do so, resulting in errors being silently swallowed). So I'd like to add some helper for `EvaluateAsync` that would just throw, and this would be the type that it'll throw.

But all places that currently catch `RException` are no-ops, and are a leftover from the time when interactions did throw. So it should be safe to remove all the catches.